fasm2bels/lib/parse_xdc.py

```diff
@@ -0,0 +1,48 @@
+""" Used to parse XDC files for pin constraints """
```

Comment:
Need to write some tests for this

Reply:
Will do.

Reply:
I was thinking something more targeted.  Like given an XDC, what constraints are generated.

Reply:
OK, sure I can add that to testing.

Reply:
FYI, this PR is just waiting on the testing.  Otherwise looks good.

Reply:
Yes, targeted testing is still on my todo.  Should be able to get to it on Friday.

Reply:
Awesome!  Just didn't want to languish forever, as this is a good change!